### ISSUE TITLE
[Critical] Input error is not described in text

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#610]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87834 |
| **Checkpoint** | 3.3 Input Assistance › Errors Identification (3.3.1.b) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 03 New Job Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/13/26 at 9:25 am |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

Input validation failures are not described in text or a text alternative.
On Packaging screen (Start a New Job, then select files, then Next), if you select Next without filling in any information, only the Package Name error appears.

## Recommendation

Rule : 
Text-based alerts that identify the field in error MUST be provided to users for form validation cues and errors.

How To Fix: 
Fix this issue by doing ALL of the following:
1. Provide an error message in text format.
2. * Include the field name in the error message and/or programmatically associate the error message and the field in error - typically using the aria-describedby attribute to reference a unique id attribute value on the error message container.
3. Describe the reason for the error, being as specific as possible.

* NOTE: Programmatic association of the error message and form field is STRONGLY recommended.

Reference: 
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/forms/notifications/

Background: 
When form fields that have an error are not identified with a text-based error message people who are blind or have low vision may not be able to understand that an error has occurred and what went wrong. In order to provide feedback that can be accessed by all users, error messages must be provided in a text format. Error messages must indicate which input is in error - either by the text of the error message or programmatically - and the nature of the error. This will allow people to recover from mistakes while filling out a form.

## Screenshots

![03_error identification](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87834/03_error_identification.png)
